CONTRIBUTING.md

```diff
@@ -26,17 +26,17 @@ Code contributions via pull requests are much appreciated. Before sending us a p
 
 1. You are working against the latest source on the *master* branch.
 2. You check existing open, and recently merged, pull requests to make sure someone else hasn't addressed the problem already.
-3. You open an issue to discuss any significant work - we would hate for your time to be wasted.
+3. You open an issue to discuss any significant work before implementing it. Major new features, submodules, and model contributions need to fit into our overall design philosophy and our user's interests. We also need to evaluate if the contribution is worth the maintenance overhead.
 
 To send us a pull request, please:
 
-1. Fork the repository.
+1. Fork the repository and clone the source code to your local machine.
 2. Modify the source (see details below); please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
 3. Ensure local tests pass.
```

Comment:
Do we have an example showing which tests need to pass?

Reply:
removed reference to tests since it would take too long for them to run